packages/react/src/patterns/blocks/ContentGroupSimple/__stories__/ContentGroupSimple.stories.js

```diff
@@ -28,6 +28,7 @@ export default {
               ? ContentGroupSimpleKnobs.mediaData.image
               : ContentGroupSimpleKnobs.mediaData.video,
           heading: text('Heading', ContentGroupSimpleKnobs.heading, groupId),
+          copy: text('Copy (optional):', ContentGroupSimpleKnobs.copy, groupId),
```

Comment:
@guilhermelMoraes can you check the knob labels in this one as well to add the actual prop names in parenthesis?